tiledbcontents/caching.py

```diff
@@ -239,9 +239,9 @@ def arrays(self):
 
 def _load_paginated(partial: Callable[..., Any]) -> List[object]:
     first_result = partial(page=1, per_page=_PER_PAGE)
-    everything = list(first_result.arrays)
+    everything = list(first_result.arrays or ())
```

Comment:
Do we want `()` or `[]`?

Reply:
it doesn’t matter in this case, because both of them are just an empty iterable. in general, though, it’s better to use the empty tuple unless you specifically need mutability.